parsers/aachen/parser.py

```diff
@@ -65,6 +66,17 @@ def add_meals_from_table(canteen, table, day):
         if category and name:
             canteen.addMeal(day, category, name, notes, prices=price_tag)
 
+            # TODO: Move determinism upstream (PyOpenMensa)
```

Comment:
Deterministic XML output should be guaranteed in PyOpenMensa, since it cannot be handled here.

Reply:
Yes, deterministic output is better ensured by PyOpenMensa.